plugins/bufbuild/protoschema-pubsub/v0.1.0/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1.6
+FROM --platform=$BUILDPLATFORM golang:1.22.1-bookworm AS build
+
+ARG TARGETOS TARGETARCH
+ENV CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH 
+
+RUN --mount=type=cache,target=/go/pkg/mod \
+    go install -ldflags="-s -w" -trimpath github.com/bufbuild/protoschema-plugins/cmd/protoc-gen-pubsub@v0.1.0
```

Comment:
I think you need to add `mv /go/bin/${GOOS}_${GOARCH}/protoc-gen-pubsub /go/bin/protoc-gen-pubsub || :` after this completes (if it exists). Try running `docker build --platform linux/amd64 .` to see the error without it.